### ISSUE TITLE
Add components overview tool like Storybook.

### ISSUE THREAD

**Issue by shapkarin** [aeternity/superhero-ui#972]
It will be good to have components overview tool like [Storybook](https://storybook.js.org). Because from time to time project is on maintenance, also that's pretty handy for the check the whole rebranding and overview the all components.
[Storybook with Vue.js blog post](https://storybook.js.org/blog/introducing-storybook-for-vue)

**Comment by davidyuk** [comment]
We have been used Storybook in the Base aepp. It was useful in developing some common components. But we were frequently accidentally breaking the components from there. So, it requires some additional maintenance.

**Comment by shapkarin** [comment]
What means `But we were frequently accidentally breaking the components from there`, how it can be Storybook fault?

**Comment by davidyuk** [comment]
I mean, making a change in a component you have to check how it looks in Storybook even if it is not necessary. Otherwise, a Storybook page may become broken and unusable.

**Comment by shapkarin** [comment]
@davidyuk what if I create some test to check that?

**Comment by shapkarin** [comment]
At least it can be a test with node.js that parse files:
- grab all components `components` property scope
- save the uniq names
- match that with Storybook file

**Comment by shapkarin** [comment]
> At least it can be a test with node.js that parse files:
> 
> * grab all components `components` property scope
> * save the uniq names
> * match that with Storybook file

but it can be overhead due some components include other sub-components as parts that doesn't mean much without parent component. so maybe it can be some manually tracked list of components...

**Comment by davidyuk** [comment]
> @davidyuk what if I create some test to check that?

I think the script can just open each storybook page in the browser and ensure that there are no exceptions.

**Comment by shapkarin** [comment]
I suggest such test with node.js:
- get all list of the pages from `src/views`
- check their "components" properties
- grab that all and uniq the array
- match it with the Storybook

**Comment by shapkarin** [comment]
> > @davidyuk what if I create some test to check that?
> 
> I think the script can just open each storybook page in the browser and ensure that there are no exceptions.

It can with Cypress.

**Comment by shapkarin** [comment]
> I suggest such test with node.js:
> 
> * get all list of the pages from `src/views`
> * check their "components" properties
> * grab that all and uniq the array
> * match it with the Storybook

But I'm still not sure how to automatically get the list of the all components. Because sometimes one component include other and some of them are useless without the parent one and I'm not sure hot to determinate that programatically.

**Comment by shapkarin** [comment]
@davidyuk the main problem with such test is to find out what components are needs to be included in the Storybook and which are pages or the parts of other components that are useless without parent component.

**Comment by davidyuk** [comment]
I don't think so. As I know, Storybook renders only pages defined by the developer, to check them script just needs to open each one. It shouldn't care about which components are referenced on these pages.

**Comment by shapkarin** [comment]
I understand that but I want to find the answer for that question: witch components actually match with the Storybook defined  by an automatically test.

**Comment by davidyuk** [comment]
Why do you need that?

**Comment by shapkarin** [comment]
> But maybe it can be checked automatically somehow.

You mean only to check if Storybook is not broken? I thought to check if all components are present at the Storybook.

**Comment by shapkarin** [comment]
So if to check only if Storybook is not broken it's pretty easy.

**Comment by shapkarin** [comment]
@davidyuk as far as I can see together we finally formalize this task in:
- add storybook
- create a test to check if Storybook is works fine itself and included components as well

I will assign this task to myself.

**Comment by davidyuk** [comment]
I'm proposing to add some pages as well. Ensure that this is approved before starting to work on it.

**Comment by shapkarin** [comment]
@davidyuk exactly that what pages is about?

**Comment by davidyuk** [comment]
Actually, they are called stories. See the example in the [Base aepp](https://github.com/aeternity/aepp-base/blob/develop/src/stories/AeButton.stories.js) repo.

**Comment by shapkarin** [comment]
@davidyuk @mradkov I think it will be nice to have such thing for the rebranding. Not sure about priority, maybe can be done later.
